app/constants/constants.js

```diff
@@ -48,18 +50,21 @@ const commonSqlOptions = [
     }
 ];
 
+const hadoopQLOptions = [
+    {'label': 'Host', 'value': 'host', 'type': 'text' },
+    {'label': 'Port', 'value': 'port', 'type': 'number'},
+    {'label': 'Database', 'value': 'database', 'type': 'text'},
```

Comment:
from https://github.com/plotly/falcon-sql-client/pull/203/files#diff-8dfc3d2e9af153a048589abb22a13822R19 it looks like `database` is optional. if so, we should add a description that says something like "database name is optional" or describe whatever it is that we do if it is not specified like "database name is not required. If database name is not specified, all tables are returned."

Reply:
Yes, and actually this is relevant for both spark and Impala -- So I am going to update it for both !!